web/src/lib/taxation/helpers.ts

```diff
@@ -0,0 +1,58 @@
+import { postTaxFilingsOnboarding } from "@/lib/api-client/apiClient";
+import { UpdateQueue } from "@/lib/types/types";
+import { getCurrentBusiness } from "@businessnjgovnavigator/shared/index";
+import { LookupLegalStructureById } from "@businessnjgovnavigator/shared/legalStructure";
+import { ProfileData } from "@businessnjgovnavigator/shared/profileData";
+const displayBusinessName = (profileData: ProfileData): boolean => {
+  return LookupLegalStructureById(profileData.legalStructureId).elementsToDisplay.has("businessName");
+};
+const displayResponsibleOwnerName = (profileData: ProfileData): boolean => {
+  return LookupLegalStructureById(profileData.legalStructureId).elementsToDisplay.has("responsibleOwnerName");
+};
+interface Props {
+  updateQueue: UpdateQueue;
+  optionalProfileDataIfSeparateContext?: ProfileData;
+}
+// the aim of this fn is to hit postTaxFilingsOnboarding and staged update queue
+export const toBeNamed = async (props: Props) => {
```

Comment:
I pulled this code down locally, and I'm not actually seeing this being used anywhere. You mentioned that there were some changes made to `TaxAccessStepTwo.tsx` and `SidebarCardFundingNudge.tsx`. I'm not seeing anything here.

Take another look at this in the morning and see if you've got something on your local machine that's missing here.